### PR TITLE
Update the Docker example README with a bit more information

### PR DESCRIPTION
**Pull request by jlvandenhout**
# Description

The current README distributed with the Docker example is quite brief. While all information is readily available on the Wiki (except for how to get the JWT to get access to your API), I though we could give a bit more information to get started properly right from the example.

## Type of change

Please delete options that are not relevant.

- [x] Documentation Fix

# How Has This Been Tested?

I went through all the steps myself to set up a node.

# Checklist:

- [x] I have selected the `develop` branch as the target branch

### CODE REVIEW

File: docker-example/README.md [hunk 1]
```diff
@@ -10,22 +28,48 @@ DASHBOARD_PASSWORD=0000000000000000000000000000000000000000000000000000000000000
...
+```
+./prepare_docker.sh
+```
 
-Change the values accordingly, then run `docker-compose up -d`
+Generate a password hash and salt for the password you want to use to access your dashboard:
 
-You can check the logs using `docker-compose logs`
+```
+docker compose run hornet tool pwd-hash
```
Comment (svenger87) [inline docker-example/README.md:33]: docker-compose

Comment (muXxer) [reply docker-example/README.md:33]: In the latest docker versions, docker compose is integrated and then you can use `docker compose`

Comment (svenger87) [reply docker-example/README.md:33]: > In the latest docker versions, docker compose is integrated and then you can use `docker compose`

Interesting. A fresh ubuntu 20.04 doesn't have that functionality yet.

Comment (centercirclesolutions) [reply docker-example/README.md:33]: https://docs.docker.com/engine/install/ubuntu/

```
curl -fsSL https://get.docker.com -o get-docker.sh
DRY_RUN=1 sh ./get-docker.sh
```

**Review by muXxer — APPROVED** [review]

**Pull request merged**